htdocs/compta/facture/fiche-rec.php

```diff
@@ -1253,15 +1254,21 @@
 
 		// Note public
 		print '<tr><td>';
-		print $form->editfieldkey($form->textwithpicto($langs->trans('NotePublic'), $htmltext, 1, 'help', '', 0, 2, 'notepublic'), 'note_public', $object->note_public, $object, $user->rights->facture->creer);
+		if ($action != 'editnote_public')
```

Comment:
We should not have any test here. Condition can be into the 5th parameter of editfieldkey.
If condition is false, behavior is already to show just the label.